app/models/manageiq/providers/azure/inventory/parser/cloud_manager.rb

```diff
@@ -484,6 +485,64 @@ def cloud_databases
     end
   end
 
+  def cloud_database_servers
+    collector.sql_servers.each do |server|
+      rg_ems_ref = collector.get_resource_group_ems_ref(server)
+
+      persister.cloud_database_servers.build(
+        :ems_ref        => server.id,
+        :name           => server.name,
+        :server_type    => 'SQL',
+        :region         => server.location,
+        :status         => server.properties&.state,
```

Comment:
@agrare Do we need some sort of normalization on the status field?  Or perhaps a raw_status vs normalized_status?